examples/routes/get.py

```diff
@@ -0,0 +1,47 @@
+from typing import Literal
+from fastapi import APIRouter
+from pydantic import BaseModel, Field
+from fastapi.responses import JSONResponse
+
+from itgs import Itgs
+from models import StandardErrorResponse
+
+router = APIRouter()
+
+
+class ExampleJobResponse(BaseModel):
+    number: int = Field(description="the random number between 1 and 10000")
+
+
+ERROR_404_TYPE = Literal["not_found"]
+"""the type of error for a 404 response"""
+
+
+@router.get(
+    "/get_example_result",
```

Comment:
this route stutters

GET /api/1/examples/get_example_result

instead it should be

GET /api/1/examples/job